ledger/controllers/wallet.js

```diff
@@ -110,9 +111,23 @@ const read = function (runtime, apiVersion) {
               $set: { unsignedTx: result.unsignedTx }
             }
           }
-          await wallets.update({ paymentId: paymentId }, state, { upsert: true })
         }
       }
+
```

Comment:
I would prefer we set `state=null` here rather then clear after use above

Reply:
why? the logical place to put it is after it is used, so if it isn't set again, we know we don't have to save it?

Reply:
my thinking is that clearing it right before use is defensive since it makes it less likely that we add an additional branch that doesn't clear it